src/background/background.js

```diff
@@ -10,7 +10,7 @@ const checkValidToken = token => token && token !== 'undefined' && token !== 'nu
 /**
  * Note: this listener can't be an async function since it must return a boolean, not a Promise<boolean>.
  */
-chrome.runtime.onMessage.addListener((req, sender, sendResponse) => {
+chrome.runtime.onMessage.addEventListener((req, sender, sendResponse) => {
```

Comment:
i don't think this is correct. this isn't a standard DOM event listener, it's part of the extension API: https://developer.chrome.com/docs/extensions/reference/api/runtime#event-onMessage

Reply:
The other method is deprecated in a newer release, that's why I changed it. But if we aren't upgrading libs then I can roll it back this evening.

Reply:
Also sorry for the late reply @vitosamson i must have missed the notification

Reply:
can you link to any information about that deprecation? i think you may be confusing this with adding a [DOM event listener](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener) which is different from the `chrome.runtime.onMessage` API for extensions

Reply:
Reverted the change, if we bump the lib version we can revisit you may be right on DOM vs chrom libs but not worried about it at the moment.